api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.8.0"
+   "version": "v1.6.0"
```

Comment:
@luxas found my mistake I forgot to run `git fetch upstream` 🤦‍♂️  maybe this should be included in the update script (or at least a check)?